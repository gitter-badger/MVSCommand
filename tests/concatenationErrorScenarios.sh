#
# Test error scenarios around concatenation
#
set -x
mvscmd --concat=UNKA:UNKB:UNKC #Error allocating non-existant datasets
mvscmd --concat=UNKA: #Error missing dataset in concatenation list
mvscmd --concat=:UNKA #Error missing dataset in concatenation list
mvscmd --concat=CEE.SCEERUN:::CEE.SCEERUN #Error missing dataset in concatenation list