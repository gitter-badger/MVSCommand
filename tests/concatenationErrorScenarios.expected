+ mvscmd --concat=UNKA:UNKB:UNKC 
Error allocating dataset UNKA
Error allocating dataset UNKB
Error allocating dataset UNKC
+ mvscmd --concat=UNKA: 
No dataset name specified for DDName concat.
+ mvscmd --concat=:UNKA 
No dataset name specified for DDName concat.
+ mvscmd --concat=CEE.SCEERUN:::CEE.SCEERUN 
No dataset name specified for DDName concat.
